app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -131,16 +133,23 @@ end
 -%>
 
 rootpw --iscrypted <%= root_pass %>
+<%# Firewall -%>
 <% if host_param_true?('disable-firewall') -%>
 firewall --disable
 <% else -%>
 firewall --<%= os_major >= 6 ? 'service=' : '' %>ssh
 <% end -%>
+<%# Kdump -%>
+<% if host_param('kdump-options').presence -%>
```

Comment:
Looks like a safe mode quirk where `.presence` isn't allowed on `nil`. You can work around it:
```suggestion
<% if host_param('kdump-options')&.presence -%>
```